extensions/firefox/content/PdfJsDefaultPreferences.sys.mjs

```diff
@@ -15,8 +15,6 @@
 
 "use strict";
```

Comment:
As far as I recall this is implied when loading something as a module, hence this line can just be removed.

Reply:
*Completely unrelated here, but:* We really should get rid of this file (and the associated mozilla-central code). Please see draft-PR #15209, which will required some feedback/help about how that should integrate properly on the mozilla-central side.

Reply:
If you turn on the `strict` rule (at least for `mjs` files), then it'll enforce removing `"use strict";` in these files.